src/XperienceCommunity.Localization.Base/Localizer/XperienceStringLocalizerBase.cs

```diff
@@ -0,0 +1,182 @@
+﻿using AngleSharp.Common;
+using CMS.ContentEngine;
+using CMS.DataEngine;
+using CMS.Helpers;
+using CMS.Websites.Routing;
+using Microsoft.Extensions.Localization;
+using System.Data;
+using XperienceCommunity.Localization;
+
+namespace XperienceCommunity.Localizer.Internal
```

Comment:
Can you check and fix this namespace?

Reply:
Made "Internal" Folder and set these under it so it now matches.